scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -235,11 +235,17 @@ namespace <%= namespace %> {
             if (StorePaymentSettings != null) {
                 callback(StorePaymentSettings, null);
             } else {
-                Client.paymentSettings((paymentSettings, error) => {
-                    if (error == null) {
-                        StorePaymentSettings = paymentSettings;
+                var query = new QueryRootQuery();
+                DefaultQueries.shop.PaymentSettings(query);
+
+                Client.Query(query, (response, error) => {
+                    if (error != null) {
+                        callback(null, error);
+                    } else {
+                        StorePaymentSettings = response.shop().paymentSettings();
+
+                        callback(response.shop().paymentSettings(), null);
```

Comment:
Nit: could probably use: `callback(StorePaymentSettings, null)` to save a few dictionary lookups